From: "Barak A. Pearlmutter" <bap@debian.org>
Date: Tue, 8 Sep 2026 13:43:24 +0100
Subject: Fix "succeded" misspelling in the download notification

Forwarded: https://codeberg.org/impromptux/ytdl-gui/issues
Last-Update: 2026-09-08

The desktop notification shown when the window is not active used the
misspelled string "Download succeded", while the message box used the
correctly spelled "Download Succeeded".  Use the latter in both places.
As that message id already exists in every translation catalogue, no
translation is invalidated by this change.
---
 src/mainactions.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mainactions.cpp b/src/mainactions.cpp
index 8a973b0..67c5d21 100644
--- a/src/mainactions.cpp
+++ b/src/mainactions.cpp
@@ -221,7 +221,7 @@ void ytdl::printResult(int result_num) {
                 if (!no_feedback && is_active) {
                     success.exec();
                 } else {
-                    system(("notify-send --icon page.codeberg.impromptux.ytdl-gui \"" + QCoreApplication::tr("Yt Downloader").toStdString() + "\" \"" + QCoreApplication::tr("Download succeded").toStdString() + "\" >/dev/null 2>&1 &").c_str());
+                    system(("notify-send --icon page.codeberg.impromptux.ytdl-gui \"" + QCoreApplication::tr("Yt Downloader").toStdString() + "\" \"" + QCoreApplication::tr("Download Succeeded").toStdString() + "\" >/dev/null 2>&1 &").c_str());
                 }
 
                 emit userAccepted();
